src/cart/crt_group.c

```diff
@@ -1022,10 +1022,7 @@ crt_grp_priv_create(struct crt_grp_priv **grp_priv_created,
 	if (grp_priv == NULL)
 		D_GOTO(out, rc = -DER_NOMEM);
 
-	rc = D_RWLOCK_INIT(&grp_priv->gp_rwlock_ft, NULL);
-	if (rc != 0)
-		D_GOTO(out_grp_priv, rc);
-
+	D_INIT_LIST_HEAD(&grp_priv->gp_sec_list);
```

Comment:
do we need to initialize this list for secondary groups? it's only meaningful in primary groups.

Reply:
Strictly speaking no, we dont need to, but all this does is just sets up pointers in the list, so no harm doing it for all groups

Reply:
ok no problem then. I thought it allocates a dummy head node.